src/gui/askforoauthlogindialog.ui

```diff
@@ -95,6 +95,13 @@
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QPushButton" name="logoutButton">
```

Comment:
We should add some icon.

Reply:
Maybe in https://github.com/owncloud/client/issues/9995